packages/js/product-editor/src/components/custom-fields/edit-modal/style.scss

```diff
@@ -9,6 +9,12 @@
 
 	.components-base-control {
 		width: 100%;
+
+		.components-input-base {
+			.components-input-control__input {
+				height: 36px;
```

Comment:
[Same](https://github.com/woocommerce/woocommerce/pull/48949/files#r1664920993) here.

Reply:
I thought we were not using those vars anymore instead we should use `$grid-unit` but I did not use it in this case since 36 does not fit any value from the grid system.